reactor-core/src/main/java/reactor/core/publisher/Mono.java

```diff
@@ -396,10 +396,11 @@
 	 * <p>
 	 * @param runnable {@link Runnable} that will be executed before emitting the completion signal
 	 *
+	 * @param <T> an arbitrary generic
```

Comment:
I'd say: "The generic type of the upstream, which is preserved by this operator"